app/services/scrape.py

```diff
@@ -79,6 +79,52 @@ def get_user_by_username(self, username) -> TwitterUser:
 
         return user_db
 
+    @cachedmethod(cache=_cache_func, key=_make_key("get_user_by_id"))
+    def get_user_by_id(self, twitter_id: str) -> TwitterUser:
```

Comment:
this one is nearly the same as `get_user_by_username()`, should we combine it with `if else`

Reply:
A little duplicate is OK